src/Common/MorganStanley.ComposeUI.Common.Interfaces/MorganStanley.ComposeUI.Common.Interfaces.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<!--
+********************************************************************************************************
+
+Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+
+********************************************************************************************************
+-->
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This package seems unnecessary, should be merged into a `MorganStanley.ComposeUI.Core` package with all the abstractions and types commonly used by all plugins.

Reply:
again, see my comment above.

Reply:
MorganStanley.ComposeUI.Core functionality should be moved under BasicModels project under a different Jira.